server/src/main/java/org/opensearch/client/IndicesAdminClient.java

```diff
@@ -838,4 +841,16 @@ public interface IndicesAdminClient extends OpenSearchClient {
      * Resolves names and wildcard expressions to indices, aliases, and data streams
      */
     ActionFuture<ResolveIndexAction.Response> resolveIndex(ResolveIndexAction.Request request);
+
+    /** Create a view */
+    void createView(CreateViewAction.Request request, ActionListener<CreateViewAction.Response> listener);
+
+    /** Create a view */
```

Comment:
thoughts on adding a little more description around the overloaded definitions of  creatView() and searchView()?

Reply:
Curious as to why we need both? That seems to be a standard though